### ISSUE TITLE
Cryptic error message as a result of bad gating fraction

### ISSUE THREAD

**Issue by castillohair** [taborlab/FlowCal#236]
When using the Excel UI, using an invalid gating fraction (e.g. 1.5) gives the following cryptic error message:

```
Sample ID S0002...
Loading file "FCFiles/data002.fcs"...
Performing data transformation...
Performing gating...
Traceback (most recent call last):
  File "C:\Users\casti\Anaconda2\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Users\casti\Anaconda2\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "d:\castillohair\documents\rice\lab\programs\flowcal\FlowCal\excel_ui.py", line 1468, in <module>
    plot=args.plot)
  File "d:\castillohair\documents\rice\lab\programs\flowcal\FlowCal\excel_ui.py", line 1400, in run
    plot_dir='plot_samples')
  File "d:\castillohair\documents\rice\lab\programs\flowcal\FlowCal\excel_ui.py", line 791, in process_samples_table
    full_output=True)
  File "d:\castillohair\documents\rice\lab\programs\flowcal\FlowCal\gate.py", line 487, in density2d
    Nidx = np.nonzero(csvH >= n)[0][0]    # we want to include this index
IndexError: index 0 is out of bounds for axis 0 with size 0
```

The error says nothing about what the problem is, and it crashes the execution of the whole program. Instead, a more descriptive error message should be printed in the "Analysis notes" column.

**Comment by castillohair** [comment]
Addressed in 81a6a9756cb71b8c1818dd4f7392deadf291e00d.

**Comment by castillohair** [comment]
Solution merged into develop, see #237.
